docs/UserGuide.md

```diff
@@ -175,6 +175,15 @@ Format: `cancelApp i/NRIC d/DATE from/STARTTIME to/ENDTIME`
 Examples:
 * `cancelApp i/ S8743880A d/ 2024-02-20 from/ 11:00 to/ 11:30`
 
+### Finding appointments: `findApp`
+
+Finds appointments based on the given parameters.
+
+Format: `findApp [i/NRIC] [d/DATE] [from/TIME]`
+
+* Filters an appointment with specific `NRIC`, `DATE` or `TIME`
+* If invalid parameters, error detailing what went wrong will be displayed.
+
```

Comment:
Maybe can provide more details on which appointments are filtered, especially the filter by time, does the command retrieve all appointments starting after the time specified or as long as the time specified falls into the range of the appointment?

Reply:
Also, just curious if providing the time without the date will still fetch me all appointments after that time for all days?